### PR TITLE
Add release notes for 20.10.4

### PR DESCRIPTION
**Pull request by tiborvass**
Signed-off-by: Tibor Vass <tibor@docker.com>

### CODE REVIEW

**Review by chris-crone — APPROVED** [review]
LGTM

File: engine/release-notes/index.md [hunk 1]
```diff
@@ -22,6 +22,66 @@ for Docker Engine.
...
 # Version 20.10
 
+## 20.10.4
+2021-02-26
+
+### Builder
+
+* Fix incorrect cache match for inline cache with empty layers [moby/moby#42061](https://github.com/moby/moby/pull/42061)
+* Update BuildKit to v0.8.2 [moby/moby#42061](https://github.com/moby/moby/pull/42061)
+  * Fix seccomp compatibility in 32bit arm allowing to build arm/v7 on arm64 machines
```
Comment (tonistiigi) [inline engine/release-notes/index.md:32]: No impact on moby

File: engine/release-notes/index.md [hunk 2]
```diff
@@ -22,6 +22,66 @@ for Docker Engine.
...
+  * resolver: avoid error caching on token fetch
+  * fileop: fix checksum to contain indexes of inputs preventing certain cache misses
+  * Update qemu emulators. Fixes a networking regression
```
Comment (tonistiigi) [inline engine/release-notes/index.md:35]: no impact on moby

File: engine/release-notes/index.md [hunk 3]
```diff
@@ -22,6 +22,66 @@ for Docker Engine.
...
+  * Fix reference count issues on typed errors with mount references (fixing `invalid mutable ref` errors)
+  * git: set token only for main remote access allowing cloning submodules with different credentials
+* Ensure blobs get deleted in /var/lib/docker/buildkit/content/blobs/sha256 after pull. To clean up old state run `builder prune` [moby/moby#42065](https://github.com/moby/moby/pull/42065)
+* Fix pull synchronization regression with error "Failed to compute cache key in newer version" [moby/moby#42049](https://github.com/moby/moby/pull/42049)
```
Comment (tonistiigi) [inline engine/release-notes/index.md:39]: ` Fix parallel pulls synchronization regression`

File: engine/release-notes/index.md [hunk 4]
```diff
@@ -22,6 +22,66 @@ for Docker Engine.
...
+* Ensure libnetwork state files do not leak [moby/moby#41972](https://github.com/moby/moby/pull/41972)
+* Fix reading context and dockerfile from stdin with BuildKit [docker/cli#2961](https://github.com/docker/cli/pull/2961)
```
Comment (tonistiigi) [inline engine/release-notes/index.md:41]: This is not at all what this PR does. Just remove it as no user impact.

File: engine/release-notes/index.md [hunk 5]
```diff
@@ -22,6 +22,66 @@ for Docker Engine.
 
 # Version 20.10
 
+## 20.10.4
+2021-02-26
+
+### Builder
+
+* Fix incorrect cache match for inline cache with empty layers [moby/moby#42061](https://github.com/moby/moby/pull/42061)
```
Comment (tonistiigi) [inline engine/release-notes/index.md:30]: `for inline cache import with`

File: engine/release-notes/index.md (hunk 2 above)
Comment (tonistiigi) [reply engine/release-notes/index.md:35]: What networking regression btw?

Comment (tiborvass) [reply engine/release-notes/index.md:35]: something with traceroute https://github.com/docker/buildx/issues/528#issuecomment-774070063

Comment (tonistiigi) [reply engine/release-notes/index.md:35]: This is completely unrelated. Embedded emulators were updated **to** 5.2, that is the version this user claims **has** the regression. But again, these are not used in moby anyway.

**Review by tonistiigi — APPROVED** [review]

**Review by cpuguy83 — COMMENTED** [review]
LGTM

**Pull request merged**